README.md

```diff
@@ -1,2 +1,99 @@
-# Exoframe Website
+### Exoframe Website
 
+<img alt="Exoframe" src="https://github.com/exoframejs/exoframe/raw/master/logo/png/exo_blue.png" width="300">
+
+> Simple Docker deployment tool
+
+[![Build Status](https://travis-ci.org/exoframejs/exoframe.svg?branch=master)](https://travis-ci.org/exoframejs/exoframe)
+[![Coverage Status](https://coveralls.io/repos/github/exoframejs/exoframe/badge.svg?branch=master)](https://coveralls.io/github/exoframejs/exoframe?branch=master)
+[![npm](https://img.shields.io/npm/v/exoframe.svg)](https://www.npmjs.com/package/exoframe)
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg?maxAge=2592000)](https://opensource.org/licenses/MIT)
+
+Exoframe is a self-hosted tool that allows simple one-command deployments using Docker.
```

Comment:
This (and pretty much everything else in this readme) should be part of index page of the website